<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TravelLine</class>
 <widget class="QWidget" name="TravelLine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>786</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Tab 1</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>GroupBox</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="3" column="1">
           <widget class="QComboBox" name="cbRate"/>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="leBookingLimit"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Date end</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Rate plan</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="cbRoomType"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Booking limit</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Room type</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QPushButton" name="btnSetRoomRate">
            <property name="text">
             <string>Set</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="EDateEdit" name="deEnd"/>
          </item>
          <item row="0" column="1">
           <widget class="EDateEdit" name="deStart"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Date start</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>386</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="3" column="2">
        <widget class="QLineEdit" name="letlPassword"/>
       </item>
       <item row="5" column="0" colspan="3">
        <widget class="QTableWidget" name="tbltlRoomTypes">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>400</height>
          </size>
         </property>
         <column>
          <property name="text">
           <string>Room type name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>API code</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Category</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Qty</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLineEdit" name="letlUrl"/>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QLabel" name="label_61">
         <property name="text">
          <string>Username</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label_59">
         <property name="text">
          <string>URL</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="label_60">
         <property name="text">
          <string>Hotel ID</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="3">
        <widget class="QWidget" name="widget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QToolButton" name="btnAddRoomType">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/images/add.png</normaloff>:/images/add.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="btnRemoveRoomType">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/images/cancel.png</normaloff>:/images/cancel.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_7">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>285</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="letlHotelID"/>
       </item>
       <item row="5" column="3">
        <widget class="QTableWidget" name="tbltlRatePlan">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>400</height>
          </size>
         </property>
         <column>
          <property name="text">
           <string>Rate plan name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>API code</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Price, AMD</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLineEdit" name="letlUsername"/>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QLabel" name="label_62">
         <property name="text">
          <string>Password</string>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QWidget" name="widget_9" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QToolButton" name="btnAddRatePlan">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/images/add.png</normaloff>:/images/add.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="btnRemoveRatePlan">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/images/cancel.png</normaloff>:/images/cancel.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_9">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>374</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EDateEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">edateedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
